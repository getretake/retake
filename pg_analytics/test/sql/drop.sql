CREATE TABLE t (a int, b text) USING parquet;
DROP TABLE t;
SELECT * FROM t;
CREATE TABLE t (a int, b text) USING parquet;
CREATE TABLE s (a int, b text);
DROP TABLE s, t;
SELECT * FROM s;
SELECT * FROM t;
