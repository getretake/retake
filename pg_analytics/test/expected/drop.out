CREATE TABLE t (a int, b text) USING deltalake;
DROP TABLE t;
SELECT * FROM t;
ERROR:  relation "t" does not exist
CREATE TABLE t (a int, b text) USING deltalake;
CREATE TABLE s (a int, b text);
DROP TABLE s, t;
SELECT * FROM s;
ERROR:  relation "s" does not exist
SELECT * FROM t;
ERROR:  relation "t" does not exist
